napari/_qt/qt_viewer.py

```diff
@@ -460,7 +460,7 @@ def on_mouse_press(self, event):
 
         layer = self.viewer.active_layer
         if layer is not None:
-            # Line bellow needed until layer mouse callbacks are refactored
+            # Line bellow needed until layer mouse callbacks are refactored!
```

Comment:
Just wanted to highlight here that at the end of the refactor those lines can be removed